src/styles/main.scss

```diff
@@ -1,3 +1,212 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
```

Comment:
in this file must be just some common styles and import other files, please clean up here